components/use-case/FeatureShowcase.vue

```diff
@@ -100,6 +100,7 @@ h2 {
   }
   .img-fluid {
```

Comment:
Is this a bootstarp class? 

If yes we should add our class and apply css to that.